SimPEG/electromagnetics/frequency_domain/sources.py

```diff
@@ -533,12 +533,14 @@ class CircularLoop(MagDipole):
 
     current = properties.Float("current in the loop", default=1.0)
 
+    N = properties.Float("number of turns in the loop", default=1.0)
```

Comment:
I don't believe a non-integer number of turns makes sense.

Also would recommend naming the property `n_turns` instead of `N` to be more descriptive

Reply:
agree. I just copy-pasted from TDEM but we might as well fix this.